MSAL/src/native_auth/network/responses/sign_in/MSALNativeAuthInternalAuthenticationMethod.swift

```diff
@@ -30,4 +30,13 @@ struct MSALNativeAuthInternalAuthenticationMethod: Decodable, Equatable {
     let challengeType: MSALNativeAuthInternalChallengeType
     let challengeChannel: String
     let loginHint: String
+
+    func toPublicAuthMethod() -> MSALAuthMethod {
+        return MSALAuthMethod(
```

Comment:
Why is this class not called MSALNativeAuthMethod? It can be use for non native also?

Reply:
There is no plan that this class can be used by the web flow. But the name with NativeAuth would be sound not appropriate because the "auth" is not part of the header, but part of the name of the class (authentication method). So, I decided to don't use "NativeAuth" in this case, to make the name of the class clearer.